src/tests/testIpAddress.cc

```diff
@@ -144,62 +142,64 @@ TestIpAddress::testInAddr6Constructor()
 void
 TestIpAddress::testSockAddrConstructor()
 {
-    struct sockaddr_in insock;
-    struct sockaddr_in outsock;
+    // sockaddr_storage containing sockaddr_in
+    {
+        struct sockaddr_storage ss = {};
+        auto *insock = reinterpret_cast<struct sockaddr_in *>(&ss);
```

Comment:
I hope this particular code will disappear due to another change request, but the idea expressed here applies to adjusted code as well.

----

If possible, please avoid never-nil pointers to local objects and reduce the difference with the current code:

```suggestion
        auto &insock = reinterpret_cast<struct sockaddr_in&>(ss);
```

Same for the other similar changes in this PR, of course.

Reply:
Good point. I will do once that other discussion is resolved.